ee/tabby-db/src/integrations.rs

```diff
@@ -86,7 +86,7 @@ impl DbConn {
         };
 
         let res = query!(
-            "UPDATE integrations SET display_name = ?, access_token = ?, api_base = ?, updated_at = DATETIME('now'), synced = false WHERE id = ? AND kind = ?;",
+            "UPDATE integrations SET display_name = ?, access_token = ?, api_base = ?, updated_at = DATETIME('now'), synced = false, error = NULL WHERE id = ? AND kind = ?;",
```

Comment:
should error only being set to null if access token is not empty？

Reply:
need to consider whether the `api_base` has been modified as well